unityfiles/Assets/Scripts/Core/GameManager.cs

```diff
@@ -115,4 +122,13 @@ public class GameManager : Singleton<GameManager> {
         UnsubscribeFromEvents();
         this.gameDataManager.SaveData();
     }
+
+    private void PlayMainMenuMusic() {
+        OnMainMenuMusic?.Invoke(this, EventArgs.Empty);
+    }
+
+    private void PlayLevel1Music() {
+        OnLevel1Music?.Invoke(this, EventArgs.Empty);
+        Debug.Log("Eventhandler called");
+    }
```

Comment:
her er det littt som @freshfish70  nevnte vi m\ ha et system som kan h\ntere game musikk (som det j'r helt fint n\) men som ogs\ m\ kunne h\ntere lyder som dinosurer og jetpack tenker du at alt f\r en event?

Reply:
Ser du bruker amerikansk tastatur, kunne du ha skrvet, aa for Å, oe for Ø og ae for Æ, sliter litt med å fortså hva du skriver :)

Reply:
jup ser i ettertid at leseligheten har tatt et betydelig hit skal prøve og huske det

Reply:
Her elton: "her er det litt som @freshfish70 nevnte, at vi må ha et system som kan håndtere game musikk (som det gjør helt fint nå) men som også må kunne håndtere lyder som dinosaurer og jetpack. Eller tenker du at alt får en event?"

Reply:
Sånn som den funker nå, så sender den ett event, dette vil også funke på helt samme måte for lydeffekter. Man ber bare audiomanageren om å spille av en .mp3 fil